### PR TITLE
Update sequential_model.py

### PR DESCRIPTION
**Pull request by airplane2230**
Comma(,) missing minor issue

### CODE REVIEW

File: guides/md/sequential_model.md [hunk 1]
```diff
@@ -261,6 +261,7 @@ model.summary()
 
 <div class="k-default-codeblock">
 ```
+WARNING:tensorflow:Please add `keras.layers.InputLayer` instead of `keras.Input` to Sequential model. `keras.Input` is intended to be used by Functional model.
```
Comment (MarkDaoust) [inline guides/md/sequential_model.md:264]: It looks like that `Input` should be an `InputLayer` would you mind fixing it at the same time?

**Review by MarkDaoust — CHANGES_REQUESTED** [review]
This is generating a warning, can you fix that too, please?

File: guides/md/sequential_model.md (hunk 1 above)
Comment (fchollet) [reply guides/md/sequential_model.md:264]: The warning will be gone in 2.6, so alternatively you could just deleted the warning by hand.

Comment (airplane2230) [reply guides/md/sequential_model.md]: As said, I manually delete it, check please!

**Review by fchollet — APPROVED** [review]

**Pull request merged**